app/controllers/middleware_server_controller.rb

```diff
@@ -124,8 +124,17 @@ def jdbc_drivers
     render :json => {
       :status => :success, :data => drivers
     }
-  rescue StandardError => err
-    render :json => {:msg => err.message}, :status => :internal_server_error
+  rescue StandardError => _err
+    render :json => {
+      :status => :internal_server_error,
+      :data   => {
+        :msg => _("Cannot connect to provider \"%{provider}\" of server \"%{server}\". Is it running?") %
+                {
+                  :provider => mw_manager.name,
+                  :server   => mw_server.name
+                }
```

Comment:
This seems fine for now since we currently do not have REST API support for Middleware.

But once we have added the Middleware APIs, and when we make start making `API.get` calls in the Middleware Angular controller, the above error handling should be moved in the angular controller as well.